linkerd/core/src/main/scala/io/buoyant/linkerd/Linker.scala

```diff
@@ -50,13 +52,20 @@ object Linker {
   case class LinkerConfig(
     namers: Option[Seq[NamerConfig]],
     routers: Seq[RouterConfig],
+    tracers: Option[Seq[TracerConfig]],
     admin: Option[Admin]
   ) {
     def mk: Linker = {
       val interpreter = namers.map(nameInterpreter)
 
-      val namersParam = Stack.Params.empty
-        .maybeWith(interpreter.map(DstBindingFactory.Namer(_)))
+      val tracerImpls = tracers.map(_.map(_.newTracer()))
+      val tracer: Tracer = tracerImpls match {
+        case Some(definedTracers) => BroadcastTracer(definedTracers)
+        case None => DefaultTracer
+      }
+
+      val params = Stack.Params.empty
+        .maybeWith(interpreter.map(DstBindingFactory.Namer(_))) + param.Tracer(tracer)
```

Comment:
I'm skeptical about this.

Do we want to be able to trace naming-related requests? Or do we want to be able to use namers in tracers.  I don't think both makes sense; and I think it's probably more important to be able to trace naming-related requests...

Need to think about this a little.